src/utils.ts

```diff
@@ -427,3 +427,14 @@ export const getNearestScrollableContainer = (
   }
   return document;
 };
+
+export const focusNearestTabbableParent = (element: HTMLInputElement) => {
```

Comment:
it should be `focusableParent`, though `focusNearestFocusableParent` sounds weird as well. Maybe just `focusNearestParent`.